### PR TITLE
Next

### PR DESCRIPTION
**Pull request by benjello**

### CODE REVIEW

File: doc/usersguide/source/architecture.rst [hunk 1]
```diff
@@ -94,14 +79,19 @@ a new function:
      * NumpyAggregate: subclass for aggregate functions. eg. count, min,
        max, std, median.
 
-  b) FunctionExpression: subclass for functions (which take one expression as
-     argument). eg. trunc, lag, duration, ...
+  b) FunctionExpr: subclass for functions. eg. trunc, lag, duration, ...
```
Comment (benjello) [inline doc/usersguide/source/architecture.rst]: @AlexisEidelman should we keep FunctionExpr or FunctionExpression ?

**Comment by benjello** [comment]
Ok @AlexisEidelman you decided to merge it on TIL/liam2/master. Should we stick to this branch for the merge. It is convenient for comparison with liam2/liam2/master.

File: doc/usersguide/source/architecture.rst (hunk 1 above)
Comment (AlexisEidelman) [inline doc/usersguide/source/architecture.rst:82]: It's a file for documentation, you can keep the most recent version of liam2

**Comment by AlexisEidelman** [comment]
Yes, the master branch was a pure fork of liam2. Maybe, we should keep the til branch as it is. The "til" branch is the Liam2 code used by Til. 
We could update the til branch with the "master" branch but it may be too demanding. Instead we cancreate a new branch "patch" from "master" and try to import the hacks of the "til" branch. The wiki should help.

**Pull request merged**